internal/worker.go

```diff
@@ -85,7 +85,9 @@ type (
 
 		// Optional: Sets the maximum number of goroutines that will concurrently poll the
 		// temporal-server to retrieve workflow tasks. Changing this value will affect the
-		// rate at which the worker is able to consume tasks from a task queue.
+		// rate at which the worker is able to consume tasks from a task queue. Due to
+		// internal logic where pollers alternate between stick and non-sticky queues, this
+		// value cannot be 1.
```

Comment:
Should mention we'll panic if it's set to 1. 

Or, consider saying "we'll just set it to 2" and not panicking

Reply:
I will say we panic. I want to panic in this situation where they explicitly set the wrong value (I'd return an error if the API supported it).